src/components/autoFocuser.js

```diff
@@ -28,24 +35,19 @@ define(["focusManager", "layoutManager"], function (focusManager, layoutManager)
         console.debug("AutoFocuser enabled");
     }
 
-    /**
-     * Create an array from some source.
-     */
-    var arrayFrom = Array.prototype.from || function (src) {
-        return Array.prototype.slice.call(src);
-    }
-
     /**
      * Set focus on a suitable element, taking into account the previously selected.
+     * @param {HTMLElement} [container] - element to limit scope
```

Comment:
I also don't think we were wrapping parameters in brackets.

Reply:
this marks optional argument